script.js

```diff
@@ -2,10 +2,60 @@ let navToggle = document.getElementsByClassName("navToggle")[0]
 let navLinks = document.getElementsByClassName("navLinks")[0]
 
 navToggle.addEventListener("click", menuDown);
-function menuDown(){
-    if(navLinks.style.display === "flex"){
-        navLinks.style.display="none"
-    }else{
-        navLinks.style.display="flex"
+function menuDown() {
+    if (navLinks.style.display === "flex") {
+        navLinks.style.display = "none"
+    } else {
+        navLinks.style.display = "flex"
     }
 }
+let TxtRotate = function (el, toRotate, period) {
+    this.toRotate = toRotate;
+    this.el = el;
+    this.loopNum = 0;
+    this.period = parseInt(period, 10) || 500;
+    this.txt = '';
+    this.tick();
+    this.isDeleting = false;
+};
+
+TxtRotate.prototype.tick = function () {
+    let i = this.loopNum % this.toRotate.length;
+    let fullTxt = this.toRotate[i];
+    if (this.isDeleting) {
+        this.txt = fullTxt.substring(0, this.txt.length - 1);
+    } else {
+        this.txt = fullTxt.substring(0, this.txt.length + 1);
+    }
+    this.el.innerHTML = '<span class="wrap">' + this.txt + '</span>';
+    let that = this;
+    let delta = 300 - Math.random() * 100;
```

Comment:
300 is magin number anti pattern here, it's not clear what it means. Check this out to get more details on this antipattern https://medium.com/@samaddico/programming-and-magic-numbers-f766e0cd1369 .
I recomend you to move this value to some variable with name which will describe why it's used.